script/cmake-ck-dev.sh

```diff
@@ -15,8 +15,8 @@ else
 fi
 
 cmake                                                                                             \
--D CMAKE_PREFIX_PATH=/opt/rocm                                                                    \
--D CMAKE_CXX_COMPILER=/opt/rocm/bin/hipcc                                                         \
+-D CMAKE_PREFIX_PATH=/opt/rocm/                                                                 \
+-D CMAKE_CXX_COMPILER=/opt/rocm/bin/hipcc                                                        \
```

Comment:
You accidentally removed whitespace so the "column" of `\` characters doesn't line up.

Consider restoring it as follows:
```suggestion
-D CMAKE_PREFIX_PATH=/opt/rocm/                                                                   \
-D CMAKE_CXX_COMPILER=/opt/rocm/bin/hipcc                                                         \
```

Reply:
sure, i ll also fix this.